pym/_emerge/BinpkgExtractorAsync.py

```diff
@@ -39,6 +46,21 @@ def _start(self):
 			self._async_wait()
 			return
 
+		decompression_binary = varexpand(shlex_split(decomp_cmd)[0], mydict=self.env)
```

Comment:
Well, I guess it's best to split after the varexpand call, in case PORTAGE_BZIP2_COMMAND contains multiple tokens.